### DISCUSSION TITLE
Sensitivity tests

### DISCUSSION THREAD

**Discussion by AaronGullickson** [lightsociologist/hirsch_test#5] [General]
@lightsociologist Can we use the issues tab here to plan out and discuss specific next steps? Alternatively we could use the discussions tab but you would need to enable it in settings. Generally, I prefer discussions for open-ended conversations and issues for specific to-do items. We can also tie these to projects to get a Trello style kanban board. In any case, its way better than email or slack. 

I think the main goal now is to do some sensitivity analyses. What do we need:

1. Separate by field/discipline somehow. I think you were dropping humanities. Note that historical studies included more than just history. I think archealogy is also dropped. I think the best approach would be to be systematic about a division here and then run the same models separately on each subset. Off the top of my head: medicine/public health, natural sciences/engineering, social sciences, humanities, professional?
2. The relationship for prop_sole at the discipline level is a bit non-linear. Would be interesting to try to fit that a bit better, maybe with polynomials.
3. I am wondering if random slope models would be helpful at all at the disciplinary level. Like trying to get at how much the gender inequality we observe varies by discipline (and the same for sole authoring). That might stretch us beyond the project goals though.

**Comment by lightsociologist** [comment]
I think you could have opened a separate issue for enabling discussion, just sayin'.

1. Yeah I was testing dropping humanities as the data pull by the original authors was focused on science. I have strong preference for keeping things as consistent with original data as possible, but was testing for narrative sake and b/c the data are pretty clearly imperfect. So, happy to keep as-is and then running sensitivity tests. I think your suggestion makes sense, but the question then becomes if we include any of that in the text (and if so how).
2. Nonlinear fit effort works for me.
3.  I think the current story seems pretty tight, but happy to entertain mroe.

**Comment by AaronGullickson** [comment]
Ok, I am happy to drop the random slope models as I think it answers a different RQ. That leaves us with two specific things to do here. I will move this over to discussion and start two real issues. 

And yes, that should have been a separate issue. Would have given us something to resolve already!

**Comment by lightsociologist** [comment]
@AaronGullickson So I built the field specific sensitivity analyses and while there are minor field differences the findings more or less hold. If there is a sensible way to visualize these effects across fields it may make for a nice secondary analysis to this short piece. The current version picks out clinical medicine to illustrate field specific effects...note that your earlier work on field effects showed that this was likely to shakeout (I believe)
